backend/src/middleware/languages/languages.ts

```diff
@@ -1,20 +1,24 @@
 import LanguageDetect from 'languagedetect'
 import { removeHtmlTags } from '../helpers/cleanHtml'
 
-const setPostLanguage = (text) => {
+const setPostLanguage = (text, defaultLanguage) => {
   const lngDetector = new LanguageDetect()
   lngDetector.setLanguageType('iso2')
-  return lngDetector.detect(removeHtmlTags(text), 1)[0][0]
+  let languages = lngDetector.detect(removeHtmlTags(text), 1)
+  if (!(Array.isArray(languages) && languages.length > 0)) {
+    languages = [[defaultLanguage, 1.0]]
```

Comment:
I don't really understand the purpose of the ``1.0`` in this line but the rest seem good for me!